quartz.layout.ts

```diff
@@ -16,6 +16,7 @@ export const sharedPageComponents: SharedLayout = {
 
 // components for pages that display a single page (e.g. a single note)
 export const defaultContentPageLayout: PageLayout = {
+  pageHeader: [],
```

Comment:
Oh I meant beforeBody should go to `pageHeader`, and `header` should go to Header like we have before

Reply:
rewrite